tests/foreman/ui/test_discoveryrule.py

```diff
@@ -51,11 +49,13 @@ def module_discovery_env(module_org, module_location):
     discovery_org.update(['value'])
 
 
-@pytest.fixture
-def manager_user(manager_loc, module_location, module_org):
-    manager_role = entities.Role().search(query={'search': 'name="Discovery Manager"'})[0]
+@pytest.fixture(scope='module')
+def manager_user(manager_loc, module_location, module_org, module_target_sat):
```

Comment:
why using manager_loc and  module_location both?

Reply:
good catch